lib/benchee/collection_data.ex

```diff
@@ -5,7 +5,9 @@ defmodule Benchee.CollectionData do
   Consists of the recorded `samples` and the statistics computed from them.
   """
 
-  defstruct [:statistics, samples: []]
+  alias Benchee.Statistics
+
+  defstruct statistics: %Statistics{}, samples: []
```

Comment:
Not sure why but I vaguely remember there was a reason why we had `statistics` be nil (likely to distinguish it not having run better) but I guess sample_size: 0 should be enough :thinking: